src/chains/models.py

```diff
@@ -113,3 +121,25 @@ def clean(self) -> None:
             raise ValidationError(
                 {"oracle_parameter": "The oracle parameter should be set"}
             )
+
+
+class Wallet(models.Model):
+    # A wallet can be part of multiple Chains and a Chain can have multiple Wallets
+    chains = models.ManyToManyField(
+        Chain, blank=True, help_text="Chains where this wallet is enabled."
+    )
+    name = models.CharField(primary_key=True, max_length=255)
+
+    def __str__(self) -> str:
+        return f"Wallet: {self.name}"
```

Comment:
Something is wrong with this PR, wasn't Wallet already merged?

Reply:
This was done on top of the branch that added the `Wallet`. Will rebase. The commit to be reviewed is 0a867b7. Since the base branch got merged now all the changes show up on this one.